fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/SearchFilter.java

```diff
@@ -42,6 +42,22 @@
         QueryType(String type) {
             this.type = type;
         }
+
+        /**
+         * Get the QueryType enum from it's search type.
+         *
+         * @param type  The type of query (i.e. "insensitive_contains")
+         *
+         * @return the enum QueryType
+         */
+        public static QueryType fromType(String type) {
```

Comment:
There is a standard way(simpler and more readable) of achieving your purpose. Example will be https://github.com/yahoo/fili/blob/master/fili-core/src/main/java/com/yahoo/bard/webservice/config/CacheFeatureFlag.java or http://grepcode.com/file_/repo1.maven.org/maven2/javax.ws.rs/jsr311-api/1.1/javax/ws/rs/core/Response.java/?v=source

Instead of `fromType`, you have 
```java
        public String getType() {
            return type;
        }
```
And you check type by having "contains".equals(QueryType.Contains.getType()). That's how we check type of enum (as the two link example does, as well)